.github/pull_request_template.md

```diff
@@ -1,5 +1,19 @@
-#### Describe the change
+#### Details
 
-<!-- Please provide an overview of the change in this PR -->
+<!-- Usually a sentence or two describing what the PR changes -->
+
+##### Motivation
+
+<!-- This can be as simple as "addresses issue #123" -->
+
+##### Context
+
+<!-- Are there any parts that you've intentionally left out-of-scope for a later PR to handle? -->
+
+<!-- Were there any alternative approaches you considered? What tradeoffs did you consider? -->
+
+#### Pull request checklist
+<!-- If a checklist item is not applicable to this change, write "n/a" in the checkbox -->
+- [ ] Addresses an existing issue: #0000
 
 > Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge. 
```

Comment:
Would it make sense to have that be a comment in the PR template given that this mostly is addressed to the PR creator, not the reviewer?

Reply:
Thank you for your review! To be clear, you are referencing line 19 (previously line 5), which reads `Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.`? 

The team agrees that this line isn't really necessary, with checks already configured to run automatically in the PR builds, so I will remove it.

If you were referring to a different line, please feel free to open a PR with your suggestion.

Reply:
Yes, I was referring to that line! Thank you for removing that line!